spec/embedded_sass.proto

```diff
@@ -98,6 +98,13 @@ message InboundMessage {
         // generating this ID and ensuring that it's unique across all importers
         // registered for this compilation.
         uint32 file_importer_id = 3;
+
+        // The [Node.js package importer], which is a built-in Package Importer
+        // with an associated `entry_point_url` that resolves `pkg:` URLs using
+        // the standards and conventions of the Node ecosystem.
+        //
+        // [Node.js package importer]: https://github.com/sass/sass/tree/main/spec/modules.md#node-package-importer
+        NodePackageImporter node_package_importer = 5;
```

Comment:
Any reason this is `5`, I see `4` is not used?

```suggestion
        NodePackageImporter node_package_importer = 4;
```

Reply:
This conflicts with `non_canonical_scheme`, see https://github.com/sass/sass/actions/runs/6628127415/job/18004541179